app/Http/Controllers/PermissionGroup/PermissionGroupController.php

```diff
@@ -47,7 +54,10 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $permissionGroup = $this->permissionGroupRepository->findById($id);
```

Comment:
$permissionGroup = $this->permissionGroupRepository->findById($id);

=>

if (! $permissionGroup = $this->permissionGroupRepository->findById($id)) {
     abort(404);
}